backend/models/Club.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const clubSchema = new mongoose.Schema({
```

Comment:
you should rename this to match the case on User.js
```suggestion
const ClubSchema = new mongoose.Schema({
```